src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/DelegatedCodeActionParams.cs

```diff
@@ -7,8 +7,11 @@
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
 
 [DataContract]
-internal class DelegatedCodeActionParams : CodeActionParams
+internal class DelegatedCodeActionParams
 {
-    [DataMember(Name = "_vs_hostDocumentVersion")]
+    [DataMember(Name = "_razor_hostDocumentVersion")]
     public int HostDocumentVersion { get; set; }
+
+    [DataMember(Name = "_razor_codeActionParams")]
+    public required CodeActionParams CodeActionParams { get; set; }
```

Comment:
Maybe i'm being paranoid here, but inheriting worries me, because if the platform provides a model that inherits from `CodeActionParams`, with some extra data, we'd lose it when we instantiated our `DelegateCodeActionParams` request. By wrapping it instead, we're immune